### PR TITLE
Change workflow and tar behavior

### PR DESCRIPTION
**Pull request by cdvr1993**
#### What does this PR do?
1. Remove verbosity of some commands, to decrease the overhead in the connection.
2. Uncompress directly to destination folder, instead of copying from temporal folder, this helps to preserve permissions.
#### Where should the reviewer start?
1. Clone the branch.
2. Do a migration from a source environment to a destination environment.
#### How should this be manually tested?
1. After the migration is done check if the destination has the .git folder and the permissions are preserved.
2. Check your temporary folder, you must not have the wordpress content decompressed.

### CODE REVIEW

File: migration.py [hunk 1]
```diff
@@ -52,9 +52,10 @@ def _init_processes_normal(self):
             self.processes.append(process.all.DestUploadTarProcess())
         self.processes.append(process.all.DestCreateDBBackupProcess())
         self.processes.append(process.all.DestCreateWPBackupProcess())
-        self.processes.append(process.all.DestDecompressWordpressProcess())
         self.processes.append(process.all.DestErasePreviousWordpressProcess())
-        self.processes.append(process.all.DestCopyWPBackupProcess())
+        # self.processes.append(process.all.DestFixPermissionsProcess())
```
Comment (cdvr1993) [inline migration.py]: Delete comments

File: process/all.py [hunk 2]
```diff
@@ -62,6 +62,23 @@ def execute(self, args, conf):
             raise Exception(stderr.read().decode('utf-8'))
 
 
+class DestFixPermissionsProcess(AbstractProcess):
```
Comment (cdvr1993) [inline process/all.py]: This class is no longer used.

File: migration.py (hunk 1 above)
Comment (chitopunk) [inline migration.py]: this is fixed

File: process/all.py (hunk 2 above)
Comment (chitopunk) [inline process/all.py]: this is fixed too

**Comment by cdvr1993** [comment]
LGTM

**Pull request merged**